new-module/vaults/README.md

```diff
@@ -0,0 +1,82 @@
+* Leverage collateral exposure in one transaction
+* Transfer your debt between vaults to avoid liquidation
+
+
+## Main features
+
+The main features of vaults are Borrow & Repay agTokens, and leverage collateral exposure. 
+
+### Borrow & Repay agTokens
+
+The main feature of vaults is the ability to **borrow** Angle stablecoins. A vault is opened when users deposit tokens as **[collateral](/new-module/glossary.md)** into a VaultManager contract. When doing so, they can choose to borrow a certain amount of agTokens. The agTokens borrowed will be minted and deposited into their wallets, for them to use however they want. 
+
+While they have an open loan, users will have to monitor their vaults' Collateral Ratio. This metric keeps track of the "health" of the vault: the value of the collateral compared to the value of the loan. If the value of the collateral with respect to the agToken decreases, the collateral ratio will go below a certain threshold and the vault can get **[liquidated](/new-module/liquidations.md)**. 
+
+At some point, users that don't need their agTokens anymore can pay bak their debt, to get back what they originally deposited as collateral. 
+
+Borrowing stablecoins is a very useful feature for users wanting to spend funds or profit from stablecoins yield, while keeping exposure to their original token. 
+
+### Leverage collateral exposure
+
+The second feature is an extension of the first one. It lets users borrow agTokens to **leverage their collateral exposure** in one transaction. When users deposit collateral to open a vault, they can choose the Leverage feature and input the amount of additional exposure they want to the collateral token, up to a certain threshold. Then, the protocol will mint agEUR, swap it against the desired collateral, deposit it back into the vault, and repeat until the correct amount has been deposited. 
+
+Similarly than in the above situation, if the value of the collateral decreases, the vault can get liquidated. 
+
+Leveraging collateral exposure is a very useful feature for users wanting to safely increase exposure to their collateral token on chain, in a reasonable way and with a medium to long term horizon. 
+
+## Details 
+
+Vaults are defined by a specific set of information: 
+- a collateral token that is deposited
+- a debt token that is borrowed
+- and a set of parameters: 
+  - [collateral factor](/glossary.md)
+  - [minting fee](/new-module/fees.md#minting-fee)
+  - [stability fee](/new-module/fees.md#stability-fee)
+  - [liquidation surcharge](/new-module/fees.md#liquidation-surcharge)
+  - [dust amount](/new-module/vaults/README.md#dust-amount)
+
+### Collateral Ratio
+
+The most defining parameter of a vault is its **Collateral Ratio**. It defines the ratio between the value of the collateral deposited, and the value of the stablecoins borrowed. 
+
+$$
+\texttt{Collateral Ratio} = \frac{\texttt{collateral value}}{\texttt{debt value}}
```

Comment:
collateral value in stablecoin

Reply:
It should be collateral's value then
Or it means "valeur collatérale" instead of "la valeur du collatéral"